src/app/Dashboard/components/DashboardComponent.js

```diff
@@ -42,49 +46,80 @@ export class DashboardComponent extends Component {
     this.setState({ user: userInfo });
     fetchUserActivites(userInfo.id);
+    loadCategories();
+  }
+
+  openModalLoginPointsHandler = () => {
+    const { logPoints } = this.state;
+    this.setState({
+      logPoints: true,
+    });
+    console.log('logpoints', logPoints);
   }
 
   render() {
     const {
       error, society, loading, pointsEarned, activitiesLogged,
     } = this.props;
-    const { user } = this.state;
+    const { user, logPoints } = this.state;
+    console.log('before click', logPoints);
     let dashboardHtml;
     if (loading) {
       dashboardHtml = <p>Loading ...</p>;
     } else if (!loading && error) {
       dashboardHtml = <p>The was an error while fetching your data. Please try again later.</p>;
     } else {
       dashboardHtml = (
-        <div className='user-dashboard'>
-          <h2 className='user-dashboard__name col-sm-12'>{user.name}</h2>
-          <div className='col-sm-12'>
-            <h3 className='user-dashboard__level'>D2</h3>
-          </div>
-          <div className='profile-overview col-sm-12'>
-            <div className='profile-overview__image' />
-            <MyStatsComponent points={pointsEarned} activities={activitiesLogged} />
-            <SocietyStatsComponent society={society} usedPoints={1508} remainingPoints={326} />
-          </div>
-          <div className='user-dashboard__actions col-sm-12'>
-            <h3 className='user-dashboard__title'>My Activities</h3>
-            <div>
-              <ButtonComponent className='button__add'>
-                <span className='fa fa-plus' />
-                <span>Log Points</span>
-              </ButtonComponent>
-              <ButtonComponent className='button__filter'>
-                <span>Filter</span>
-                <span className='fa fa-filter' />
-              </ButtonComponent>
+        <div className='wrapper'>
+          <HeroComponent />
+          <div className='main-content'>
+            <SidebarComponent className='sidebar' />
+            <div className='sub-content'>
+              <NavbarComponent />
+              <div className='user-dashboard'>
+                <h2 className='user-dashboard__name col-sm-12'>{user.name}</h2>
+                <div className='col-sm-12'>
+                  <h3 className='user-dashboard__level'>D2</h3>
+                </div>
+                <div className='profile-overview col-sm-12'>
+                  <div className='profile-overview__image' />
+                  <MyStatsComponent points={pointsEarned} activities={activitiesLogged} />
+                  <SocietyStatsComponent usedPoints={1508} remainingPoints={326} />
+                </div>
+                <div className='user-dashboard__actions col-sm-12'>
+                  <h3 className='user-dashboard__title'>My Activities</h3>
+                  <div>
+                    {/* <ButtonComponent className='button__add'>
+                      <span className='fa fa-plus' />
```

Comment:
This PR does not seem complete. Too much commented out code and console.logs

Reply:
@kimani-ndegwa I have updated the PR and removed all the unnecessary code.